Microsoft.Toolkit.Win32.UI.Controls/DesignerUI.resx

```diff
@@ -0,0 +1,219 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@normesta  @mcleanbyron Most of what you want to review that is displayed to the user (error message, designer descriptions, designer categories, etc.) is in here